internal/command/commands/service.go

```diff
@@ -64,32 +69,72 @@ func (s *Service) Run(args []string) int {
 			return 0
 		}
 
-		configData, err := file.Read(*filePath)
+		if exists, err := dir.Exists(*directoryPath); !exists || err != nil {
+			s.Logger.Error("Provided directory path : " + *directoryPath + " , is not valid\n" + err.Error())
+			return 1
+		}
+
+		serviceDefinition, serviceDefinitionPath, err := file.FindAndReadAllAllowedFormat(*directoryPath+"/definition", []string{".json", ".yml", ".yaml"})
 		if err != nil {
-			s.Logger.Error("Unable to read from " + *filePath + "\n" + err.Error())
+			s.Logger.Error("Unable to read from " + *directoryPath + "/definition.json\n")
 			return 1
 		}
 
-		var parsedConfig interface{}
+		// Throw error on empty service def file
+		if len(serviceDefinition) == 0 {
+			s.Logger.Error("service definition file(definition.json) cannot be empty")
+			return 1
+		}
+		parsedServiceDefinition, err := utils.ParserYmlOrJson(serviceDefinitionPath, serviceDefinition)
+		if err != nil {
+			s.Logger.Error(err.Error())
+			return 1
+		}
 
-		if strings.Contains(*filePath, ".yaml") || strings.Contains(*filePath, ".yml") {
-			err = yaml.Unmarshal(configData, &parsedConfig)
+		envTypes, err := envClient.EnvTypes()
+		if err != nil {
+			s.Logger.Error(err.Error())
+			return 1
+		}
+
+		allFiles, err := dir.SubDirs(*directoryPath)
+		if err != nil {
+			s.Logger.Error(err.Error())
+			return 1
+		}
+		envFileMap := make(map[string]string)
+		r, _ := regexp.Compile(`provisioning-([a-zA-Z]*)\.json`)
+		for _, file := range allFiles {
+			envType := r.FindStringSubmatch(file)
```

Comment:
There should be a warning message saying ignoring `provisioning-non_exisitng_env_type.json` as env type does not exist